### ISSUE TITLE
"Request timed out."

### ISSUE THREAD

**Issue by superspeedie** [rawxbee/neopets-scripts#3]
Hi, I have been receiving the "Request timed out." error upon clicking my NP balance for quite a few days now. I realize it does this sometimes then starts working again, but it has yet to work - I cleared my cache and cookies, completely uninstalled and reinstalled the script multiple times, checked everything I could with my minimal knowledge of script coding (everything has since been restored, I didn't change much), but unfortunately I am still facing the same issue.

I even tried adding in the full http and https links as opposed to the */www.neopets etc etc @include lines but sadly that did not fix it either (again, I have since reverted to the original script itself, and only tried this when I noticed it had not worked for a few days).

**Comment by rawxbee** [comment]
This was happening when browsing the site through the https version rather than the http version. I believe I have fixed it so that it works on both versions of webpages (the ajax request just required that "/premium" be "/premium/" instead). Let me know if the updated version doesn't work.

**Comment by superspeedie** [comment]
Hi, thank you for looking in to this! Sadly I am still facing the same issue, though now it is taking much longer to load ("Getting data" message), but that part may be because the site is slow at the moment. I tried updating what I already had installed first, and after trying that, I uninstalled and completely reinstalled. Unfortunately it is still not working on my end. I appreciate you taking the time to look in to it, and happy holidays!
